detekt.yml

```diff
@@ -107,7 +107,7 @@ complexity:
   CyclomaticComplexMethod:
     active: true
     threshold: 15
-    ignoreSingleWhenExpression: false
+    ignoreSingleWhenExpression: true
```

Comment:
**question**: why is this needed?

Reply:
Because of `CyclomaticComplexMethod` error caused by when statement with multiple clauses by detekt :
`<ID>CyclomaticComplexMethod:LicenceStatus.kt$LicenceStatus.Companion$fun lookupLicenceEventByStatus(status: LicenceStatus): LicenceEventType</ID>`

[https://github.com/detekt/detekt/issues/680]